pages/linux/lvextend.md

```diff
@@ -1,6 +1,8 @@
 # lvextend
 
 > Increase the size of a logical volume.
+> One of the Logical Volume Manager (LVM) tools.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/logical_volume_manager_administration/index>.
```

Comment:
Nice!  But this URL is rather lengthy 😕 

How about this, to shorten it a tiny bit? (also RHEL 7 since it's the newest that that page is available one):
```suggestion
> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/logical_volume_manager_administration>.
```